gothemis/session/session_test.go

```diff
@@ -223,6 +223,29 @@ func testSession(keytype int, t *testing.T) {
 	<-finCh
 }
 
+func testInvalidKeyHandling(t *testing.T) {
```

Comment:
Since this test is not parameterized, it can be just standalone `TestInvalidKeyHandling`, without invoking it from TestSession.